code/modules/admin/admin.dm

```diff
@@ -10,6 +10,15 @@
 
 
 ///////////////////////////////////////////////////////////////////////////////////////////////Panels
+//TMentor
+/proc/is_mentor(client/C)
+	if(!istype(C))
+		return FALSE
+	if(!C.holder)
+		return FALSE
+	if(C.holder.rank.rights == R_MENTOR)

```

Comment:
Yea, that's for only mentors. To check if the person is a mentor, not just any admin.